dm/tests/dmctl_command/run.sh

```diff
@@ -6,7 +6,7 @@ cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 source $cur/../_utils/test_prepare
 WORK_DIR=$TEST_DIR/$TEST_NAME
 
-help_cnt=45
+help_cnt=46
```

Comment:
there's no integration test for the commands in this pr, maybe add subtask in #4426 and state it in the pr description

Reply:
I've added a subtask and reverted this change.